src/Chapter4.hs

```diff
@@ -629,7 +658,7 @@ Can you implement a monad version of AND, polymorphic over any monad?
 🕯 HINT: Use "(>>=)", "pure" and anonymous function
 -}
 andM :: (Monad m) => m Bool -> m Bool -> m Bool
-andM = error "andM: Not implemented!"
+andM b1 b2 = b1 >>= \b -> if b then b2 else pure False
```

Comment:
Very clean solution! :sparkles:

Reply:
I was having doubts here. I came to this solution because of the hint you left on the line above, but I was actually thinking of using a `do` block, or something like this. This bind + lambda + if/else/then in the same line made my head spin :sweat_smile: